src/sass/disability-benefits.scss

```diff
@@ -662,6 +662,12 @@ h1:focus {
   outline: none;
 }
 
+h2.claim-h2 {
```

Comment:
Good call, I can remove the `h2` selector. I remember running into a problem with `h`tags being overwritten by the uswds defaults, but it works in this case. 👍